### PR TITLE
Added trained models over 20 epochs after which performance stalls; fixed dropout behavior during training

### PR DESCRIPTION
**Pull request by balbok0**

### CODE REVIEW

File: model_training/train.py [hunk 1]
```diff
@@ -144,6 +144,7 @@ def train(args):
         ########################################################################
         # Training
         ########################################################################
+        model.train()
```
Comment (keyan) [inline model_training/train.py:147]: Ah, oops

**Comment by keyan** [comment]
Looks good thanks!
